src/main/java/redis/clients/jedis/UnifiedJedis.java

```diff
@@ -237,6 +237,17 @@ private UnifiedJedis(CommandExecutor executor, ConnectionProvider provider, Comm
     this.graphCommandObjects.setBaseCommandArgumentsCreator((comm) -> this.commandObjects.commandArguments(comm));
   }
 
+  /**
+   * Needed for unit tests.
+   */
+  public UnifiedJedis(CommandObjects commandObjects, GraphCommandObjects graphCommandObjects,
+                      CommandExecutor executor, ConnectionProvider provider) {
```

Comment:
Do you need this only for graph commands tests? If so, ignore. Graph commands are deprecated and we don't want to add complexity just for testing graph commands.